lib/compatibility_mode_gen.go

```diff
@@ -1,6 +1,5 @@
 // Code generated by "enumer -type=CompatibilityMode -transform=snake -trimprefix CompatibilityMode -output compatibility_mode_gen.go"; DO NOT EDIT.
```

Comment:
this file likely shouldn't be changed or linted

Reply:
Resolved by [this force-push](https://github.com/grafana/k6/pull/2995/commits/683ebf000dbf66fefdcfc0c6ae0c39949d3cbd67), but now I am having some troubles with the new forked enumer... :sweat_smile:

Reply:
Actually missed some things that I fixed with the next [force-push](https://github.com/grafana/k6/compare/683ebf000dbf66fefdcfc0c6ae0c39949d3cbd67..d901c1f2fdcb78a611aead13386016e033b43527)